contracts/contracts/authorization/GatedMint.sol

```diff
@@ -24,7 +24,11 @@ contract GatedMint {
     redemptions[issuer] = true;
   }
 
-  function getIssuer(MintPermit calldata permit) private view returns (address) {
+  function getIssuer(MintPermit calldata permit)
+    private
+    view
+    returns (address)
+  {
```

Comment:
why has this formatting change? New prettier version?

Reply:
Hum... no clue. No new version, I just ran the formatting task over all sol files, so I believe this file was committed unformatted.